src/commonMain/kotlin/kotools/types/collection/NotEmptySet.kt

```diff
@@ -55,6 +58,49 @@ public fun <E> Collection<E>.toNotEmptySet(): Result<NotEmptySet<E>> =
         }
         ?: Result.failure(EmptyCollectionException)
 
+/**
+ * Representation of sets that contain at least one element of type [E].
```

Comment:
This documentation should be aligned with [the `NotEmptyList` type's one][kotools.types.collection.NotEmptyList].

[kotools.types.collection.NotEmptyList]: https://github.com/kotools/types/blob/e7bb042839d7f71f78804c4198299211a5eb06aa/src/commonMain/kotlin/kotools/types/collection/NotEmptyList.kt#L64

```suggestion
 * Representation a set with at least one element of type [E].
```

Reply:
Resolved in the commit c47e0a70c2c1e6e4d7ea00ffa5c0e0803390f2eb.